src/inverted-index.js

```diff
@@ -1,42 +1,138 @@
+/**
+ * The Inverted Index class
+ * @author Babatunde Adeyemi <tundewrites@gmail.com>
+ * @class
+ */
 class InvertedIndex {
+
+  /**
+   * Initializes indices to an empty object
+   * @constructor
+   */
   constructor() {
     this.indices = {};
   }
 
-  isValid(jsonFile) {
+  /**
+   * Checks for valid JSON file
+   * @param {object} jsonFile - the JSON file to be checked
+   * @returns {boolean} - either true or false
+   */
+  static isValid(jsonFile) {
     return Array.isArray(jsonFile) && !!jsonFile.length;
   }
 
-  isMalformed(jsonFile) {
+  /**
+   * Checks for malformed JSON file
+   * @param {object} jsonFile - the JSON file to be checked
+   * @returns {boolean} - either true or false
+   */
+  static isMalformed(jsonFile) {
     let result = false;
     for (let i = 0; i < jsonFile.length; i += 1) {
       result = !('title' in jsonFile[i] && 'text' in jsonFile[i]);
```

Comment:
I don't know what (i) stands for in your code, you could replace (i) with a word that say what is really is.